jakartaee9/src/web-common_4_0.xsds

```diff
@@ -972,6 +972,19 @@
           </xsd:documentation>
         </xsd:annotation>
       </xsd:element>
+      <xsd:element name="same-site"
+                   type="jakartaee:cookie-sameSiteType"
+                   minOccurs="0">
+        <xsd:annotation>
+          <xsd:documentation>
+
+            Specifies whether any session tracking cookies created
+            by this web application will be marked with a Same-Site
+	    attribute
```

Comment:
There's some indentation or space/tab confusion here.

Reply:
My editor put a tab in there!  fixed